server/routerlicious/packages/services-utils/src/auth.ts

```diff
@@ -33,6 +33,18 @@ import {
 } from "@fluidframework/server-services-telemetry";
 import { getBooleanFromConfig, getNumberFromConfig } from "./configUtils";
 
+interface IKeylessTokenClaims extends ITokenClaims {
+	/**
+	 * Identifies if the token is for Keyless Access or not.
+	 */
+	fluidRelayKeylessAccess?: boolean;
```

Comment:
I think the type should just be `true`, and not optional. Otherwise, if `false` or `undefined`, it's just `ITokenClaims`. Instead of changing all the types to `IKeylessTokenClaims`, you can leave them _as-is_, because `IKeylessTokenClaims` _is_ a valid `ITokenClaims`. Then, internally *where it matters*, use a type guard like the following to check if it's keyless

```ts
function isKeylessTokenClaims(claims: ITokenClaims): claim is IKeylessTokenClaims {
  return "isKeylessToken" in claims && claims.isKeylessToken === true;
}
```

Reply:
Good point! Wil do this